Gemfile

```diff
@@ -3,6 +3,6 @@ source "http://rubygems.org"
 gem 'progress', :git => 'git://github.com/djstrong/progress.git'
 gem 'slop', "~> 3.6.0"
 gem 'cycr', '~> 0.2.7'
-gem 'wiktionary'
+gem 'wiktionary', :path => '../wiktionary/'
```

Comment:
Ok, to simplify thinks a present, we can use this solution. But in future, we should stick to github repos in such cases and remember to fix such "hack" when making release of the system.